include/swift/Reflection/Records.h

```diff
@@ -110,6 +110,8 @@ enum class FieldDescriptorKind : uint16_t {
   Protocol,
   ClassProtocol,
   ObjCProtocol,
+  ObjCClass,
```

Comment:
Can you use ObjCClass for Swift-defined classes that inherit from NSObject, too? (I believe IRGen has utility functions to get the reference counting style without you walking the inheritance chain directly; just map Unknown to ObjCClass).